Library/Homebrew/formula_auditor.rb

```diff
@@ -796,6 +796,8 @@ def audit_specs
         tag ||= formula.stable.specs[:tag]
 
         if @online
+          return if owner.nil? || repo.nil?
```

Comment:
I wonder if things like could/should be `T.must`

Reply:
I would think not and if the regexp doesn't match then it never gets to this code. But I thought `T.must` [was frowned upon in favour of `nil` checks](https://github.com/Homebrew/brew/pull/17606#issuecomment-2202231742), or is it just in some cases?

Reply:
I don't think it's as black and white as "never use `T.must`". In this case I'd argue `T.must` is beneficial as it'll throw a `TypeError` if it is nil and the code is written in a way that expects it to be not nil.

An example of where `T.must` might be smell is if something is genuinely nilable but we don't check it in a Sorbet-friendly way:

```rb
# where var.method returns T.nilable(something)

# bad
return if var.method.nil?

T.must(var.method).something

# good
return if (value = var.method).nil?

value.method
```

In this case however it can never be nil so `T.must` IMO seems appropriate rather than a false nil check.

Reply:
Alright, thanks, done!